store/kvtest.cc

```diff
@@ -0,0 +1,26 @@
+#include <optional>
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include "kvstore_client.h"
+
+DEFINE_string(store, "", "Stores to the given file name");
+DEFINE_string(key, "", "Stores key to kvstore");
+DEFINE_string(value, "", "Stores value to kvstore");
+
+int main(int argc, char** argv) {
```

Comment:
You may want to define some tests and use EXPECT_EQ() or otherwise to check the outcomes of the tests.

Reply:
Will try